core/Db/Schema/Mysql.php

```diff
@@ -334,6 +334,24 @@ public function getTablesCreateSql()
                                       ) $tableOptions
             ",
 
+            'archiving_metrics'     => "CREATE TABLE {$prefixTables}archiving_metrics (
+                                      metadataid BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                      idarchive BIGINT UNSIGNED NULL,
+                                      idsite INTEGER UNSIGNED NOT NULL,
+                                      segment VARCHAR(255) NULL,
+                                      date1 DATE NOT NULL,
+                                      date2 DATE NOT NULL,
+                                      period VARCHAR(10) NOT NULL,
```

Comment:
```suggestion
                                      period TINYINT UNSIGNED NOT NULL,
```

We usually never store a period in a format other than the internal numeric identifier.